### ISSUE TITLE
Broken provider icon in Dashboard

### ISSUE THREAD

**Issue by strarsis** [containous/traefik#5813]
### What did you do?
Open a weighted service in Dashboard (HTTP Services tab).

### What did you expect to see?
Proper provider icons in Services list.

### What did you see instead?
Broken image indicator from browser instead of a provider icon.
![dc4d5fcd-f3a8-4632-88cc-0af2e227dbf8](https://user-images.githubusercontent.com/9271436/68625854-67424780-04da-11ea-9d74-2ce66adfe5fe.png)

### Output of `traefik version`: (_What version of Traefik are you using?_)
```
Version:      2.0.4
Codename:     montdor
Go version:   go1.13.3
Built:        2019-10-28T20:23:57Z
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?
`docker-compose.yml`:
```yml
version: '3'

services:
  reverse-proxy:
    image: traefik:v2.0.4
    container_name: "traefik"
    restart: always
    command:
      #- "--log.level=DEBUG"
      - "--api.insecure=true"
      - "--providers.docker=true"
      - "--providers.docker.exposedbydefault=false"
      - "--providers.file.filename=/etc/traefik/dynamic_conf.toml"
      - "--providers.file.watch=true"
      - "--entrypoints.web.address=:80"
      - "--entrypoints.websecure.address=:443"
      - "--certificatesresolvers.myhttpchallenge.acme.httpchallenge=true"
      - "--certificatesresolvers.myhttpchallenge.acme.httpchallenge.entrypoint=web"
      #- "--certificatesresolvers.myhttpchallenge.acme.caserver=https://acme-staging-v02.api.letsencrypt.org/directory"
      - "--certificatesresolvers.myhttpchallenge.acme.email=webmaster@example.com"
      - "--certificatesresolvers.myhttpchallenge.acme.storage=/letsencrypt/acme.json"
    ports:
      - "80:80"
      - "443:443"
      - "8080:8080"
    volumes:
      - ./letsencrypt:/letsencrypt
      - ./conf/dynamic_conf.toml:/etc/traefik/dynamic_conf.toml
      - /var/run/docker.sock:/var/run/docker.sock
    networks:
      - web

````

File provider for extra config:
```toml
[http.routers]
  [http.routers.test1]
    rule = "Host(`example.com`)"
    entrypoints = [ "websecure" ]
    service = "test1-api"
    [http.routers.test1.tls]
      certresolver = "myhttpchallenge"

[http.services]
  [http.services.test1-api]
    [[http.services.test1-api.weighted.services]]
      name = "test1-apiv1"
      weight = 1

  [http.services.test1-apiv1]
    [http.services.test1-apiv1.loadBalancer]
      [[http.services.test1-apiv1.loadBalancer.servers]]
        url = "127.0.0.2"
```

**Comment by ldez** [comment]
Hello,

I use your configuration and I don't reproduce:
![Screenshot from 2019-11-12 00-26-30](https://user-images.githubusercontent.com/5674651/68629212-33b7eb00-04e3-11ea-9cbb-943034f90f09.png)

**Comment by strarsis** [comment]
@ldez: You have to open the weighted service (`test1-api@file`).

Side note: It would be nice if one can just click on the listed services that are listed in another service and then go to that service instead. Currently one has to go back to services list first, then select the service that has been previously listed for the other service.

**Comment by ldez** [comment]
Ok I reproduce:
![Screenshot from 2019-11-12 01-45-09](https://user-images.githubusercontent.com/5674651/68632259-2b18e200-04ee-11ea-8488-544ea4d02ad9.png)

**Comment by traefiker** [comment]
Closed by #5983.
